app/assets/stylesheets/arclight/modules/hierarchy_and_online_contents.scss

```diff
@@ -22,10 +22,6 @@ $hierarchy-view-collapse-icon: url("data:image/svg+xml,<svg xmlns='http://www.w3
   }
 }
 
-.al-hierarchy-highlight > .documentHeader {
```

Comment:
I would leave this in -- I think it is useful to visually highlight the currently selected item. And now that you have unlinked the current highlighted item, its text color is black so there's no contrast issue to resolve on the highlighted document.